### DISCUSSION TITLE
A working blog using Apéro

### DISCUSSION THREAD

**Discussion by johngoldin** [apreshill/blogophonic#83] [Show and tell]
I now have a [working version of my site](https://admiring-swanson-66e6d4.netlify.app/blog/) in apéro. It has quite a few rough edges, but a lot of the rough edges were there even before I did the conversion. It looks much nicer than my [current site](https://www.johngoldin.com) and that may motivate me to polish the content a bit. I especially like the way it looks on my iPhone. So far I haven't done anything with Collections or Projects, although that's a big part of the reason why I was motivated to change themes. I very much like the way a series of blog posts came out. I have one [here](https://admiring-swanson-66e6d4.netlify.app/blog/walking-in-britain/am-walker-1/), but there are two other series I plan to have later.

I understand that apéro may still be a work in progress and that I shouldn't count on things not changing, but it has been interesting to try to use it this soon in its life.

My biggest problem doing the conversion had nothing to do with apéro. The first line in my config file is baseURL. In my old config file that was my full URL of my site, complete with http. When viewing the new site locally that caused no problem at all. Everything looked fine. But in Netlify it caused weird problems. I lost the apéro css stuff so fonts and icons were off. I took me an embarrassingly long time before I realized the problem was that I needed to switch baseURL to /, at least for now.

I'm still trying to figure out good practice with images to dress things up. Am I correct that there's no way to use the header yaml in an unbundled blog post to point to an image? I don't think the `images` item does anything and I don't think I can use a png with "feature" in the name unless it's a bundle. I'll probably bundle up some of my old posts just so that I can add an image.

I have put in Hugo aliases in the headers of the blog posts and that seems to take care of redirection from the URLs in my old site (or it will once I switch over to my permanent domain name). I thought I couldn't make Netlify redirects work, but that was probably just another symptom of my screw-up with baseURL.

I have a lot more I can work on, but the progress so far has been very satisfying.

**Comment by muhammetozkaraca** [comment]
I am not sure whether you were able to find out a solution to this problem, which I have a similar one when I change the baseURL to http, I lost the images on the website while I could view them locally, I am glad that you post how to resolve the problem momentarily, by changing baseURL to /. Thank you for that, because most probably, I could not figure out that on my own to be honest with you.
